core/src/main/scala/kafka/admin/AclCommand.scala

```diff
@@ -345,11 +346,11 @@ object AclCommand extends Logging {
       .describedAs("delegation-token")
       .ofType(classOf[String])
 
-    val resourceNameType = parser.accepts("resource-name-type", "The type of the resource name, or any.")
+    val resourcePatternType = parser.accepts("resource-pattern-type", "The type of the resource pattern, or any.")
```

Comment:
We want to add MATCH in the description and explain its meaning.

Reply:
Added more detail